src/domain/usecases/GetAllSurveysUseCase.ts

```diff
@@ -11,30 +11,50 @@ export class GetAllSurveysUseCase {
     constructor(private surveyReporsitory: SurveyRepository) {}
 
     public execute(
-        surveyType: SURVEY_FORM_TYPES,
-        parentSurveyId: Id | undefined
+        surveyFormType: SURVEY_FORM_TYPES,
+        orgUnitId: Id,
+        parentPPSSurveyId: Id | undefined,
+        parentWardRegisterId: Id | undefined
     ): FutureData<Survey[]> {
-        let orgUnitId = "";
-        const programId = getProgramId(surveyType);
+        const programId = getProgramId(surveyFormType);
 
         //All PPS Survey Forms are Global.
-        if (surveyType === "PPSSurveyForm") orgUnitId = GLOBAL_OU_ID;
+        if (surveyFormType === "PPSSurveyForm") orgUnitId = GLOBAL_OU_ID;
```

Comment:
Maybe here it's better to create a new variable instead of change the current variable `orgUnitId`: `const newVariableName = surveyFormType === "PPSSurveyForm" ? GLOBAL_OU_ID : orgUnitId`

Reply:
done in https://github.com/EyeSeeTea/amr-surveys/pull/11